ui/api/consumerGroups/schema.ts

```diff
@@ -2,6 +2,11 @@ import { ApiError } from "@/api/api";
 import { NodeSchema } from "@/api/kafka/schema";
 import { z } from "zod";
 
+const ConsumerGroupStateSchema = z.union([
+  z.literal("STABLE"),
+  z.literal("EMPTY"),
```

Comment:
Let's list all of the possibilities (as of Kafka 3.8) to avoid unexpected errors validating the schema.
```suggestion
  z.literal("UNKNOWN"),
  z.literal("PREPARING_REBALANCE"),
  z.literal("COMPLETING_REBALANCE"),
  z.literal("STABLE"),
  z.literal("DEAD"),
  z.literal("EMPTY"),
  z.literal("ASSIGNING"),
  z.literal("RECONCILING"),
```

Reply:
Done